README.md

```diff
@@ -110,6 +135,7 @@ aws serverlessrepo put-application-policy \
 ## Install Lambda into AWS
 
 ### Sceptre
+
```

Comment:
can you add some docs around which account this lambda should be installed to?  also you might want to let admins know that [moving SES out of the sandbox](https://docs.aws.amazon.com/ses/latest/dg/request-production-access.html) is a requirement for sending emails.

Can you also please update the existing docs this section as well?

Reply:
yeah, I can update that

Reply:
done